apps/web/assets/js/components/Filters.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+
+const toggleLevels = function(levels, level) {
+  if (levels.includes(level)) {
+    var index = levels.indexOf(level)
+    levels.splice(index, 1)
+  } else {
+    levels.push(level)
+  }
+
+  return levels
+}
+
+export default class Filters extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.toggleAdvanced = this.toggleAdvanced.bind(this)
+    this.toggleIntermediate = this.toggleIntermediate.bind(this)
+    this.toggleStarter = this.toggleStarter.bind(this)
+  }
+
+  badgeCSS(level) {
+    if (this.props.filters.levels.includes(level)) {
+      return `badge--${level}`
+    } else {
+      return 'badge--disabled'
+    }
+  }
+
+  toggleStarter() {
```

Comment:
Depending on what ES6 stages you're supporting, you can use instance method notation here and avoid having to `.bind(this)` on your methods. Here's an example:

```
class SomeClass {
  instanceMethod = () => {
    // `this` won't be bound until an instance of this class is created
    this.doSomething();
  }

  nonInstanceMethod() {
    // Since this method lives on the prototype, you would need to bind `this` if you try to call
    // it from a different context
    this.doSomething();
  }
}
```

If your build pipeline supports transpiling this, then you could drop the constructor altogether and just define the toggle methods using the `method = () => ` syntax.

Reply:
Based on the fact that I see `babel-latest` and `babel-react` in your brunch config, I'm guessing instance methods are supported out of the box.